### ISSUE TITLE
[a11y]: Select - Playground - JAWS is not announcing the  warn state of the combo box

### ISSUE THREAD

**Issue by guidari** [carbon-design-system/carbon#16614]
### Package

@carbon/react

### Browser

Firefox

### Package version

v1.58.0

### React version

18.02

### Automated testing tool and ruleset

Manual QA

### Assistive technology

JAWS

### Description

When the warn Boolean set to "True", warn icon will display on the combo box input field accordingly. Observe JAWS is not announcing about the warn state of the combo box.

### Recommend Fix
Ensure JAWS should announce the warn state of the combo box when the Boolean is set to "True"

### Reproduction/example

https://react.carbondesignsystem.com/?path=/story/components-select--playground&args=warn:!true;readOnly:!false

### Steps to reproduce

none

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by heloiselui** [comment]
Since this playground example doesn’t exist in our Storybook, we’re not able to reproduce or verify the issue as described. I believe we can close this issue.
